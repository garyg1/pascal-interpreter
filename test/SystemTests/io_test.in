5
-1.23E+13
