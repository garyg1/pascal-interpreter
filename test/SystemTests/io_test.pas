program io_test;

Var
    a : double;

begin
    Readln(a);
    Writeln(a);

    Readln(a);
    Writeln(a);
end.