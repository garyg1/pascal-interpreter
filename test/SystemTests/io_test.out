5.0
-1.23e13
